### DISCUSSION TITLE
Set not working in explicit many-to-many relation

### DISCUSSION THREAD

**Discussion by nabilsarder** [prisma/prisma#13044] [Q&A]
Not getting any errors, but no records are being updated when using set. Does set work with explicit many to many relations? This link https://www.prisma.io/docs/support/help-articles/working-with-many-to-many-relations provides an example for implicit but not explicit relations.

**Comment by nurul3101** [comment]
Hey @nabilsarder 👋 

Could you please share the schema file and query which you are invoking, so that I could take a look?

**Comment by nabilsarder** [comment]
```prisma
// Schema
model Blank {
  id String @id @default(cuid())
  resellers ResellersOnBlanks[]
}

model Reseller {
  id String @id @default(cuid())
  blanks ResellersOnBlanks[]
}

model ResellersOnBlanks {
  assignedAt DateTime @default(now())
  assignedBy String?
  blankId    String
  resellerId String
  blank      Blank    @relation(fields: [blankId], references: [id])
  reseller   Reseller @relation(fields: [resellerId], references: [id])


  @@id([blankId, resellerId])
}
```

```javascript
// Query
return await db.blank.update({
  where: { id: blankId },
  data: {
    resellers: {
      set: [...data],
    },
  },
})
```

No errors invoking this query, but no change in db.

**Reply by pantharshit00** [reply]
Hello @nabilsarder, our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

I wasn't able to reproduce this with the current version. Can you make sure this is still reproducible?

**Reply by pantharshit00** [reply]
Hello again @nabilsarder, I am going through past discussions right now and saw that this is still open. Can you please confirm if this is still relevant to you?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
